src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php

```diff
@@ -85,8 +85,9 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $output = new SymfonyStyle($input, $output);
```

Comment:
It was already mentioned here https://github.com/symfony/symfony/pull/14132#discussion_r27491835 but I'm not sure about it